README.md

```diff
@@ -1,12 +1,12 @@
 HTTPRequests
 ============
-A Block that makes HTTP requests.  One request is made for every input signal. For each successful request, an output signal is created with the url that was hit.
+The HTTPReqeusts block sends an HTTP request for each item in the list of incoming signals. For each successful request, an output signal is created that includes the response from the request.
```

Comment:
Fix typo in block name. 

Is "item" ambiguous here? What is the best way to describe signals, lists of signals, and streams of signals? I think this change is good, but as we go through, let's try to come up with a clear and consistent way to refer to these things.